src/fluree/db/ledger/transact/error.clj

```diff
@@ -111,9 +115,11 @@
           hash-flake       (tx-meta/generate-hash-flake flakes tx-state)
           all-flakes       (conj flakes hash-flake)
           fast-forward-db? (:tt-id db)
-          db-after         (if fast-forward-db?
-                             (<? (dbproto/-forward-time-travel db all-flakes))
-                             (<? (dbproto/-with-t db all-flakes)))
+          db-after         (->> (if fast-forward-db?
+                                  (<? (dbproto/-forward-time-travel db all-flakes))
+                                  (<? (dbproto/-with-t db all-flakes)))
+                                dbproto/-rootdb
+                                tx-util/make-candidate-db)
```

Comment:
Again here, use db-root from the tx-state instead of db-before (which is permissioned). Then you can remove dbproto/-rootdb and tx-util/make-candidate-db.

Reply:
This appears the be resolved for 2 or the 3 instances.  In throw-validation-exception, we are creating our own tx-state context to handle any unknowns.